src/server/session.ts

```diff
@@ -2196,6 +2201,38 @@ namespace ts.server {
             });
         }
 
+        private toggleLineComment(args: protocol.FileRangeRequestArgs): TextChange[] {
+            const { file, project } = this.getFileAndProject(args);

```

Comment:
These need to use `getFileAndLanguageServiceForSyntacticOperation`

Reply:
Changed.